DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -1430,8 +1430,31 @@ globus_l_gfs_iRODS_command(
             }
             break;
 
+        case GLOBUS_GFS_CMD_RNTO:
+            {
+                globus_gfs_log_message(GLOBUS_GFS_LOG_INFO,"iRODS: GLOBUS_GFS_CMD_RNTO\n");
+
+                char * from_path = strdup(cmd_info->from_pathname);
+                if (from_path == nullptr)
+                {
+                    result = GlobusGFSErrorGeneric("iRODS: strdup failed");
+                    globus_gridftp_server_finished_command(op, result, GLOBUS_NULL);
+                    break;
+                }
+                iRODS_l_reduce_path(from_path);
+                globus_gfs_log_message(GLOBUS_GFS_LOG_INFO,"rename from %s to %s\n", from_path, collection);
+
+                dataObjCopyInp_t dataObjRenameInp;
+                dataObjRenameInp.srcDataObjInp.oprType = dataObjRenameInp.destDataObjInp.oprType = RENAME_DATA_OBJ;  
```

Comment:
Is this normal to assign two things at once?

Reply:
It's supported by C++ for sure.

Reply:
People often do it as the assignment operator returns the value itself.  I copied this from other iRODS code but can change it if preferred.

Reply:
no worries.   just caught my eye and makes the line long.   either is fine.

Reply:
There is some trailing whitespace on this line, too.